files/brews/freetype.rb

```diff
@@ -3,15 +3,15 @@
 class Freetypephp < Formula
   homepage 'http://www.freetype.org'
   url 'http://downloads.sf.net/project/freetype/freetype2/2.4.11/freetype-2.4.11.tar.gz'
-  sha1 'a8373512281f74a53713904050e0d71c026bf5cf'
+  sha256 '29a70e55863e4b697f6d9f3ddc405a88b83a317e3c8fd9c09dc4e4c8b5f9ec3e'
 
   keg_only "Sandboxed for PHP installations"
 
   option :universal
 
   bottle do
     # Included with X11 so no bottle needed before Mountain Lion.
-    sha1 '7dc4747810e51beb99fd36c8f5baade6e65d19b7' => :mountain_lion
+    sha256 'TODO_Figure_this_out' => :mountain_lion
```

Comment:
I'm not sure how many versions of Mac boxen supports but we could remove this entire section (at least until someone using Mac before Mountain Lion needs support and thus can help provide the SHA 256 hash).